internal/bootstrap/bootstrap_plain_git.go

```diff
@@ -329,10 +302,13 @@ func (b *PlainGitBootstrapper) ReconcileSyncConfig(ctx context.Context, options
 
 	// Apply to cluster
 	b.logger.Actionf("applying sync manifests")
-	kubectlArgs := []string{"apply", "-k", filepath.Join(b.git.Path(), filepath.Dir(kusManifests.Path))}
-	if _, err = utils.ExecKubectlCommand(ctx, utils.ModeStderrOS, b.kubeconfig, b.kubecontext, kubectlArgs...); err != nil {
+	applyOutput, err := utils.Apply(ctx, b.kubeconfig, b.kubecontext, filepath.Join(b.git.Path(), kusManifests.Path))
+	if err != nil {
 		return err
 	}
+
+	fmt.Fprintln(os.Stderr, applyOutput)
```

Comment:
Given this package is located outside of `cmd`, I think hardcoding `os.Stderr` here instead of relying on some configured value is a (leaked) assumption.

Reply:
I have removed it as it makes the bootstrap output look really messy and it's useless at this step.